src/main/webapp/definitions/alfresco-core.yaml

```diff
@@ -2123,6 +2123,39 @@ paths:
           description: Unexpected error
           schema:
             $ref: '#/definitions/Error'
+    delete:
+      x-alfresco-since: "7.1.1"
+      tags:
+        - renditions
+      summary: Delete rendition
+      description: |
+        **Note:** this endpoint is available in Alfresco 7.1.1 and newer versions.
+
+        Delete the rendition identified by **renditionId** of file **nodeId**.
+
+        If the rendition is successfully deleted then the content for that rendition node
+        will be cleared.
+      operationId: deleteRendition
+      parameters:
+        - $ref: '#/parameters/nodeIdParam'
+        - $ref: '#/parameters/renditionIdParam'
+      responses:
+        '204':
+          description: Successful response
+        '400':
+          description: |
+            Invalid parameter: **nodeId** is not a valid format, or is not a file, or **renditionId** is invalid
```

Comment:
Same here - is not a file sounds like an upload failure.

Reply:
done